go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/cosmos/cosmos-sdk/db v1.0.0-beta.1.0.20220726092710-f848e4300a8a
 	github.com/cosmos/go-bip39 v1.0.0
 	github.com/cosmos/gogogateway v1.2.0
-	github.com/cosmos/gogoproto v1.4.4
+	github.com/cosmos/gogoproto v1.4.5-0.20230213172703-83dec24dea1d
```

Comment:
In this file, we use the `google.golang.org/protobuf v1.28.2-0.20230208135220-49eaa78c6c9c`. It was updated by last week dependabot in #14958, indirectly via `grpc`.

Each time we have an innocent dependabot update, there's a non-zero risk that the underlying .*proto files are also changed (even if it's just updating comments), causing file descriptors mismatch.

This means that gogoproto will need to be updated in lockstep with `google.golang.org/protobuf` changes _on their main branch_. That adds a huge maintenance burden on gogoproto.

Reply:
I'm rethinking if file descriptor equality is needed. Maybe if there's a mismatch, let's just take the protoregistry one.

Reply:
Logging a warning now.